api/v1/customTypes/int12.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function (mongoose) {
+    function Int12(key, options) {
+        mongoose.SchemaType.call(this, key, options, 'Int12');
```

Comment:
I think it's really better to rename that "uint12" for the future users / developers to understand correctly that we can't have negative values